components/config/src/config/markup.rs

```diff
@@ -45,6 +45,8 @@ pub struct Markdown {
     pub external_links_no_referrer: bool,
     /// Whether smart punctuation is enabled (changing quotes, dashes, dots etc in their typographic form)
     pub smart_punctuation: bool,
+    /// Whether to enable latex math rendering
+    pub math: bool,
```

Comment:
We want that to be an enum. Eventually (hopefully) there will be math rendering inside Zola (eg https://zola.discourse.group/t/a-simpler-katex-alternative-that-works-today/2156/3) so we will want to support classes/pre-render etc